README.md

```diff
@@ -155,7 +176,7 @@ connection.execute(query: query, parameters: supplied_key1, supplied_key2) { que
 
```

Comment:
As I can't comment in the SQL Injection Prevention using Parameterization section here is some new text to replace the existing text in that section:

"Unsanitized data that is used in dynamic queries is one of the most common causes of SQL injection vulnerabilities. Parameterizing queries can help to prevent SQL injection attacks."

"The following code is vulnerable to SQL injection if `supplied_key1` or `supplied_key2` contain untrusted data (that is, data which has not been validated):"

"To reduce the possibility of SQL injection attacks, use the following parameterised version of the code:"